examples/PACKAGES/mbx/2048h2o/in.mbx_h2o

```diff
@@ -0,0 +1,58 @@
+processors      * * * map xyz
```

Comment:
See my comments on the previous example.

Reply:
Why do we need four different size water examples with identical inputs?
Just use the smallest and be done with it.